go-controller/pkg/crd/egressfirewall/v1/register.go

```diff
@@ -0,0 +1,25 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	GroupName     = "ovnkubernetes.ovn.org"
```

Comment:
for consistency with our annotations this should probably be "`k8s.ovn.org`"?

Reply:
changed the group name to k8s.ovn.org